### ISSUE TITLE
initctl

### ISSUE THREAD

**Issue by totto** [altran/Whydah-Provisioning#4]
initctl restart UserIdentityBackend
initctl: Rejected send message, 1 matched rules; type="method_call", sender=":1.4" (uid=501 pid=29969 comm="initctl restart UserIdentityBackend ") interface="com.ubuntu.Upstart0_6.Job" member="Restart" error name="(unset)" requested_reply="0" destination="com.ubuntu.Upstart" (uid=0 pid=1 comm="/sbin/init ")

**Comment by totto** [comment]
The provisioning should handle such situations :)

**Comment by StigLau** [comment]
Doesn't this happen because the app was started manually by someone not using the restart script on the ec2-user?

**Comment by totto** [comment]
It probably happens because it find wrong PID or PID/user combinantion

**Comment by StigLau** [comment]
correct. Initctl can't control UserIdentityBackend if it didn't start it and has the PID

**Comment by totto** [comment]
pids=$(pgrep <name>)
will get you the pids of all processes with the given name. To kill them all, use

kill -9 $pids
To refrain from using a variable and directly kill all processes with a given name issue

pkill -9 <name>
